create_jenkins_job.py

```diff
@@ -159,7 +159,7 @@ def main(argv=None):
     os_config_overrides = {
         'linux-rhel': {
             'mixed_overlay_pkgs': '',
-            'ignore_rmw_default': {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp', 'rmw_connextdds'},
+            'ignore_rmw_default': data['ignore_rmw_default'] | {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp', 'rmw_connextdds'},
```

Comment:
Nit, but if you are doing this you can also remove the hard-coded `rmw_connext_dynamic_cpp` (it's in the 'ignore_rmw_default' list already):

```suggestion
            'ignore_rmw_default': data['ignore_rmw_default'] | {'rmw_connext_cpp', 'rmw_connextdds'},
```